src/renderer/api-decorator.js

```diff
@@ -427,7 +427,7 @@ limitations under the License.
         let webContentsId = getWebContentsId();
         // Reset state machine values that are set through synchronous handshake between native WebContent lifecycle observers and JS
         options.openfin = true;
-        options.preload = options.preload || winOpts.preload;
+        options.preload = 'preload' in options ? options.preload : winOpts.preload;
```

Comment:
A bit confusing because `winOpts` is not in fact the options for the new child window (which is in `options`)... unless `winOpts` is a standard usage (is it?), let's please take this opportunity to rename `winOpts` to `mainWindowOptions` (or `appOpts`?)